entity_api/src/organization.rs

```diff
@@ -5,17 +5,14 @@ use sea_orm::{entity::prelude::*, ActiveValue};
 use serde_json::json;
 use service::AppState;
 
-pub async fn find_all(app_state: &AppState) -> Vec<Model> {
+pub async fn find_all(app_state: &AppState) -> Result<Vec<Model>, Error> {
     let db = app_state.database_connection.as_ref().unwrap();
-    Entity::find().all(db).await.unwrap_or(vec![])
+    Ok(Entity::find().all(db).await?)
 }
 
 pub async fn find_by_id(app_state: &AppState, id: i32) -> Result<Option<Model>, Error> {
     let db = app_state.database_connection.as_ref().unwrap();
-    Entity::find_by_id(id)
-        .one(db)
-        .await
-        .map_err(|err| err.into())
+    Ok(Entity::find_by_id(id).one(db).await?)
```

Comment:
I like this a lot. I'm curious, though, if we're now only transforming the error up one level in the actual controller? I think we'd need to write a test to determine this.

Reply:
I think you figured this out better in your other branch. Wrapping in `Ok()` is a valid solution because the `Result<>` is unwrapped from await(), but we need it to be in a `Result<>` for both the `Ok` or `Err` cases.

But having a naked `?` operator is always going to be my favorite solution syntactically.